src/app/beer_garden/api/http/handlers/v1/job.py

```diff
@@ -210,11 +217,145 @@ async def post(self):
+        responses:
+          201:
+            description: All new jobs have been created
+            schema:
+              $ref: '#/definitions/JobImport'
+          400:
+            $ref: '#/definitions/400Error'
+          50x:
+            $ref: '#/definitions/50xError'
+        tags:
+          - Jobs
+        """
+        empty_body_regex = re.compile(r"\s*\{\s*\}\s*")  # noqa
+        decoded_body: str = self.request.body.decode("utf-8")
+
+        if not len(decoded_body) or empty_body_regex.match(decoded_body):
+            raise ValueError("Body was empty")
+
+        parsed_job_dfn_list: Optional[JobDefinitionList]
+        parsed_job_dfn_list = SchemaParser.parse_job_definitions(
+            decoded_body, from_string=True
+        )
+
+        if parsed_job_dfn_list is None or parsed_job_dfn_list.jobs is None:
+            raise ValueError("Could not parse any job definitions")
+
+        cleaned_job_dfn_list = JobDefinitionList(
+            list(map(self._clean_job, parsed_job_dfn_list.jobs))
+        )
+
+        response = await self.client(
+            Operation(operation_type="JOB_CREATE_MULTI", args=[cleaned_job_dfn_list])
+        )
+
+        self.set_status(201)
+        self.set_header("Content-Type", "application/json; charset=UTF-8")
+        self.write(response)
+
+    @classmethod
+    def _clean_job(cls, job: brewtils.models.Job) -> brewtils.models.Job:
+        """Remove fields from a Job definition that a new Job would not have."""
+        job_copy = deepcopy(job)
+
+        job_copy.error_count = 0
+        job_copy.id = None
+        job_copy.next_run_time = None
+        job_copy.success_count = 0
+
+        return job_copy
+
+
+class JobExportAPI(BaseHandler):
+    def data_received(self, chunk: bytes) -> Optional[Awaitable[None]]:
+        return super().data_received(chunk)
+
+    @authenticated(permissions=[Permissions.CREATE])
+    async def post(self):
+        """
+        ---
+        summary: Exports a list of Jobs from a list of IDs.
+        description: |
+          Jobs will be scheduled from a provided list to run on the intervals
+          set in their trigger arguments.
+        parameters:
+          - name: jobs
+            in: body
+            description: A list of the Jobs IDs whose job definitions should be \
+            exported. Omitting this parameter will export all jobs.
+            schema:
+              $ref: '#/definitions/JobImport'
+        responses:
+          201:
+            description: A list of jobs has been exported.
+            schema:
+              $ref: '#/definitions/JobExport'
+          400:
+            $ref: '#/definitions/400Error'
+          50x:
+            $ref: '#/definitions/50xError'
+        tags:
+          - Jobs
+        """
+        filter_params_dict = {}
+
+        # we're focused on the "ids" list, but if arguments are passed, allow
+        # them through; that is, provide maximum flexibility to the user
```

Comment:
This sounds like we're providing a secret, undocumented API.  Meaning the schema definition tells you that you can use "ids", but we'll secretly take other arguments too.  I'm not sure that's a thing we want to do.

Reply:
Removed.